dataikuapi/dss/ml.py

```diff
@@ -741,6 +823,44 @@ def get_computation_params(self):
         computation_params["randomState"] = self.get("randomState")
         computation_params["onSample"] = self.get("onSample")
         return computation_params
+    
+    def list_modalities(self):
+        """
+        List definitions of modalities
+        """
+        return [m.definition for m in self.modalities]
+
+    def get_modality(self, definition=None, missing_values=False):
+        """
+        Retrieves modality from definition
+
+        :param definition: definition of modality to retrieve. Can be:
+                   * :class:`dataikuapi.dss.ml.DSSSubpopulationModalityDefinition`
+                   * for category modality, can be a str corresponding to the value of the modality
+                   * for numeric modality, can be a number inside the modality
+        :param missing_values: whether to retrieve modality corresponding to missing values. If True, 
+                               `definition` is ignored
```

Comment:
Alternative option to consider: replace `missing_value` argument by a call with `definition= DSSSubpopulationModalityDefinition.MISSING_VALUES` (being a constant Definition)

Reply:
Yeah that's better, thx